developer/examples/build.vpc.centos.pkr.hcl

```diff
@@ -46,7 +46,8 @@ source "ibmcloud-vpc" "centos" {
   security_group_id = var.SECURITY_GROUP_ID
 
   vsi_base_image_name = "ibm-centos-7-9-minimal-amd64-5"
-
+    
+  //vsi_boot_volume_id = ""
```

Comment:
Would we plan to provide sample use case ?

Reply:
added a new packer template file with this use case. developer/examples/build.vpc.centos.boot.volume.pkr.hcl